From 49f3edbc3f0f0cfbbde57efe88be4cbf921bf779 Mon Sep 17 00:00:00 2001
From: Sebastian Rasmussen <sebras@gmail.com>
Date: Fri, 20 Feb 2015 19:21:28 +0100
Subject: 744865 Fix typo in singular form of string
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

    * src/persp/dbgperspective/nmv-call-stack.cc
      (CallStack::Priv::append_frames_to_tree_view): Fix typo in
      singular form of string

Signed-off-by: Marek Černocký <marek@manet.cz>
---
 src/persp/dbgperspective/nmv-call-stack.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/persp/dbgperspective/nmv-call-stack.cc b/src/persp/dbgperspective/nmv-call-stack.cc
index 68b80d1..b79a9f6 100644
--- a/src/persp/dbgperspective/nmv-call-stack.cc
+++ b/src/persp/dbgperspective/nmv-call-stack.cc
@@ -765,7 +765,7 @@ struct CallStack::Priv {
             store_iter = store->append ();
             UString msg;
             msg.printf (ngettext ("(Click here to see the next %d row of the "
-                                  "call stack",
+                                  "call stack)",
                                   "(Click here to see the next %d rows of the "
                                   "call stack)",
                                   nb_frames_expansion_chunk),
-- 
cgit v0.12


From 7005393a8c4d914eac9705e7f47818d0f4de3578 Mon Sep 17 00:00:00 2001
From: Marcin Kolny <marcin.kolny@gmail.com>
Date: Wed, 7 Oct 2015 12:16:49 +0200
Subject: build fix

---
 src/confmgr/nmv-gconf-mgr.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/confmgr/nmv-gconf-mgr.cc b/src/confmgr/nmv-gconf-mgr.cc
index 672935b..e212ff6 100644
--- a/src/confmgr/nmv-gconf-mgr.cc
+++ b/src/confmgr/nmv-gconf-mgr.cc
@@ -32,6 +32,7 @@
 NEMIVER_BEGIN_NAMESPACE (nemiver)
 
 using nemiver::common::GCharSafePtr;
+using nemiver::common::GErrorSafePtr;
 
 class GConfMgr : public IConfMgr {
     GConfMgr (const GConfMgr &);
-- 
cgit v0.12

From 0ad382f79c6249fe0a46121d04fca0a2044bcfc0 Mon Sep 17 00:00:00 2001
From: Ben Iofel <iofelben@gmail.com>
Date: Thu, 17 Mar 2016 18:28:02 -0400
Subject: [PATCH] Fix compiliation warnings & errors

---
 src/dbgengine/nmv-dbg-common.h        | 2 +-
 src/dbgengine/nmv-i-var-list-walker.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/dbgengine/nmv-dbg-common.h b/src/dbgengine/nmv-dbg-common.h
index ad3cc00..0edac7c 100644
--- a/src/dbgengine/nmv-dbg-common.h
+++ b/src/dbgengine/nmv-dbg-common.h
@@ -171,7 +171,7 @@ public:
 
     bool has_slot () const
     {
-        return m_slot;
+        return static_cast<bool> (m_slot);
     }
 
     template<class T>
diff --git a/src/dbgengine/nmv-i-var-list-walker.h b/src/dbgengine/nmv-i-var-list-walker.h
index b719c0d..f2f3229 100644
--- a/src/dbgengine/nmv-i-var-list-walker.h
+++ b/src/dbgengine/nmv-i-var-list-walker.h
@@ -22,7 +22,7 @@
  *
  *See COPYRIGHT file copyright information.
  */
-#ifndef __NMV_VAR_LIST_WALKER_H__
+#ifndef __NMV_I_VAR_LIST_WALKER_H__
 #define __NMV_I_VAR_LIST_WALKER_H__
 
 #include "nmv-i-var-walker.h"
-- 
2.7.3

From 262cf9657f9c2727a816972b348692adcc666008 Mon Sep 17 00:00:00 2001
From: Marcin Kolny <marcin.kolny@gmail.com>
Date: Fri, 1 Jul 2016 19:45:05 +0200
Subject: Use RefPtr::bool() operator in the conditions

Since bool() operator in RefPtr class is explicit,
comparision with integer doesn't compile.
---
 src/persp/dbgperspective/nmv-dbg-perspective.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/persp/dbgperspective/nmv-dbg-perspective.cc b/src/persp/dbgperspective/nmv-dbg-perspective.cc
index be652db..ac207e4 100644
--- a/src/persp/dbgperspective/nmv-dbg-perspective.cc
+++ b/src/persp/dbgperspective/nmv-dbg-perspective.cc
@@ -5672,7 +5672,7 @@ DBGPerspective::switch_to_asm (const common::DisassembleInfo &a_info,
     a_source_editor->clear_decorations ();
 
     Glib::RefPtr<Gsv::Buffer> asm_buf;
-    if ((asm_buf = a_source_editor->get_assembly_source_buffer ()) == 0) {
+    if (!(asm_buf = a_source_editor->get_assembly_source_buffer ())) {
         SourceEditor::setup_buffer_mime_and_lang (asm_buf, "text/x-asm");
         a_source_editor->register_assembly_source_buffer (asm_buf);
         asm_buf = a_source_editor->get_assembly_source_buffer ();
@@ -5720,7 +5720,7 @@ DBGPerspective::switch_to_source_code ()
 
     Glib::RefPtr<Gsv::Buffer> source_buf;
     UString source_path;
-    if ((source_buf = source_editor->get_non_assembly_source_buffer ()) == 0) {
+    if (!(source_buf = source_editor->get_non_assembly_source_buffer ())) {
         // Woops!
         // We don't have any source code buffer. Let's try hard to get
         // the source code corresponding to the current frame. For that,
-- 
cgit v0.12
